Sources/SwiftUICharts/BarChart/Shapes/RoundedRectangleBarShape.swift

```diff
@@ -45,10 +45,10 @@ internal struct RoundedRectangleBarShape: Shape {
         let h = rect.size.height
         
         // Make sure we do not exceed the size of the rectangle
-        let tr = min(min(self.tr, h/2), w/2)
```

Comment:
As the comment above suggests, part of what is here is to control the size of the radius.

![Screenshot 2022-04-02 at 08 04 54](https://user-images.githubusercontent.com/63838770/161371416-aba14fe1-337e-4b06-8b80-1892e8cb1d74.png)

Without the `/2` there can be drawing errors.